staging/src/k8s.io/apimachinery/pkg/apis/meta/v1alpha1/types.go

```diff
@@ -30,28 +33,33 @@ type Table struct {
 	// Standard list metadata.
 	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#types-kinds
 	// +optional
-	v1.ListMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+	v1.ListMeta `json:"metadata,omitempty"`
 
 	// columnDefinitions describes each column in the returned items array. The number of cells per row
 	// will always match the number of column definitions.
-	ColumnDefinitions []TableColumnDefinitions `json:"columnDefinitions"`
+	ColumnDefinitions []TableColumnDefinition `json:"columnDefinitions"`
 	// rows is the list of items in the table.
 	Rows []TableRow `json:"rows"`
 }
 
-// TableColumnDefinitions contains information about a column returned in the Table.
+// TableColumnDefinition contains information about a column returned in the Table.
 // +protobuf=false
-type TableColumnDefinitions struct {
+type TableColumnDefinition struct {
 	// name is a human readable name for the column.
 	Name string `json:"name"`
 	// type is an OpenAPI type definition for this column.
 	// See https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#data-types for more.
 	Type string `json:"type"`
-	// format is an optional OpenAPI type definition for this column.
+	// format is an optional OpenAPI type definition for this column. The 'name' format is applied
```

Comment:
We should probably change it so that things like this are quoted, single ticks or back ticks, so that is clear. Doesn't have to be in this PR of course.